hapi-utils/src/main/java/module-info.java

```diff
@@ -11,6 +11,7 @@
     requires com.fasterxml.jackson.databind;
     requires com.google.protobuf;
     requires com.swirlds.common;
+    requires com.sun.jna;
```

Comment:
Can you please merge develop in your branch. This is already changed in develop.